src/trayicon.cpp

```diff
@@ -129,6 +129,23 @@ void TrayIcon::unreadCounterUpdate( unsigned int total, QColor color )
         }
     }
     
+    // Execute the hook process
+    if ( !BirdtrayApp::get()->getSettings()->mProcessRunOnCountChange.isEmpty() )
+    {
+        QString cmdline = BirdtrayApp::get()->getSettings()->mProcessRunOnCountChange;
+
+        // Replace the %NEW% with the new unread count
+        cmdline.replace( "%NEW%", QString::number( total ) );
+
+        // Replace the %OLD% with the old unread count
+        cmdline.replace( "%OLD%", QString::number( mUnreadCounter ) );
```

Comment:
What do you think about adding %DIFF% as well? Should be really easy to add and might make it easier for some commands.